### DISCUSSION TITLE
Android app keeps logging out

### DISCUSSION THREAD

**Discussion by rdar-lab** [community/community#73371] [Mobile]
I freequently use the app but it is killing me that every few days I have to login again. Please fix this.

**Comment by spotype** [comment]
Response #1

**Reply by abdul-musabbir** [reply]
asdf

**Comment by venturozzaccio** [comment]
+1, super annoying to deal with. At least have an option to leave it logged for more time.
My bank account is protected with fingerprint and doesn't need constant logins, not sure why Github mobile should try to be safer.
